packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -326,67 +323,57 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
-		[
-			{
-				eventName: "fluid:telemetry:Container:Request_cancel",
-				error: "Non interactive/summarizer client's data object should not be initialized",
-			},
-		],
-		async () => {
-			const dataStoreFactory1 = new DataObjectFactory(
-				"@fluid-example/test-dataStore1",
-				TestDataObject1,
-				[],
-				[],
-			);
-			const registryStoreEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
-				[dataStoreFactory1.type, Promise.resolve(dataStoreFactory1)],
-			]);
-			const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore({
-				defaultFactory: dataStoreFactory1,
-				registryEntries: registryStoreEntries,
-			});
-
-			// Create a container for the first client.
-			const container1 = await provider.createContainer(runtimeFactory);
-			await assert.doesNotReject(
-				requestFluidObject<TestDataObject1>(container1, "/"),
-				"Initial creation of container and data store should succeed.",
-			);
+	it("full initialization of data object should not happen by default", async () => {
+		const dataStoreFactory1 = new DataObjectFactory(
+			"@fluid-example/test-dataStore1",
+			TestDataObject1,
+			[],
+			[],
+		);
+		const registryStoreEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
+			[dataStoreFactory1.type, Promise.resolve(dataStoreFactory1)],
+		]);
+		const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore({
+			defaultFactory: dataStoreFactory1,
+			registryEntries: registryStoreEntries,
+		});
 
-			// Create a summarizer for the container and do a summary shouldn't throw.
-			const createSummarizerResult = await createSummarizerFromFactory(
-				provider,
-				container1,
-				dataStoreFactory1,
-				undefined,
-				ContainerRuntimeFactoryWithDefaultDataStore,
-				registryStoreEntries,
-			);
-			await assert.doesNotReject(
-				summarizeNow(createSummarizerResult.summarizer, "test"),
-				"Summarizing should not throw",
-			);
+		// Create a container for the first client.
+		const container1 = await provider.createContainer(runtimeFactory);
+		await container1.getEntryPoint();
+
+		// Create a summarizer for the container and do a summary shouldn't throw.
+		const createSummarizerResult = await createSummarizerFromFactory(
+			provider,
+			container1,
+			dataStoreFactory1,
+			undefined,
+			ContainerRuntimeFactoryWithDefaultDataStore,
+			registryStoreEntries,
+		);
+		await assert.doesNotReject(
+			summarizeNow(createSummarizerResult.summarizer, "test"),
+			"Summarizing should not throw",
+		);
 
-			// In summarizer, load the data store should fail.
-			await assert.rejects(
-				requestFluidObject<TestDataObject1>(createSummarizerResult.container, "/"),
-				(e: Error) =>
-					e.message ===
-					"Non interactive/summarizer client's data object should not be initialized",
+		// In summarizer, load the data store should fail.
+		try {
+			const runtime = (createSummarizerResult.summarizer as any).runtime as ContainerRuntime;
```

Comment:
nit: Maybe put these 3 lines in a const block and use assert.rejects? It just looks cleaner.